libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/trip/session/MapboxTripSession.kt

```diff
@@ -58,7 +58,7 @@ import kotlinx.coroutines.withContext
  *
```

Comment:
Let's remove the `TODO` ☝️

Reply:
This can be moved out of the `internal` package as it has the `internal` modifier now.

Reply:
fixed